app/src/main/res/values/styles.xml

```diff
@@ -1,7 +1,7 @@
 <resources>
 
     <!-- Base application theme. -->
-    <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
+    <style name="AppTheme" parent="Theme.AppCompat.Light.NoActionBar">
```

Comment:
why did you change this to no action bar 
for the main activity it is good if there is no action bar but please add the toolbar to the home feed activity

Reply:
Okay.